janusgraph-berkeleyje/src/main/java/org/janusgraph/diskstorage/berkeleyje/BerkeleyJEStoreManager.java

```diff
@@ -67,6 +67,11 @@
             "The BDB record lock mode used for read operations",
             ConfigOption.Type.MASKABLE, String.class, LockMode.DEFAULT.toString(), disallowEmpty(String.class));
 
+    public static final ConfigOption<String> CACHE_MODE =
+        new ConfigOption<>(BERKELEY_NS, "cache-mode",
+            "Modes that can be specified for control over caching of records in the JE in-memory cache",
+            ConfigOption.Type.MASKABLE,  String.class, CacheMode.DEFAULT.toString(), disallowEmpty(String.class));
```

Comment:
It would be good to know all possibilities here.

Reply:
I don't think this possibilities will be useful, because it is internal modes of bdb and for understanding it required consult with javadoc